### ISSUE TITLE
Make humhub autonomous / disable e-mail verification

### ISSUE THREAD

**Issue by mitya12342** [humhub/humhub#3191]
I am going to use humhub in local network without internet access. My main problem now is disabling email verification. Is it already implemented? What else should i do to make humhub fully autonomous?
There was similar issue #858 but i have no ways to receive e-mail.

**Comment by ArchBlood** [comment]
@luke- could this be closed due to the following being implemented or is this function something separate from what's being asked for?

https://github.com/humhub/humhub/blob/498a457a809879470de039d3aa757179c5876734/protected/humhub/modules/user/Module.php#L100-L104

**Comment by luke-** [comment]
@ArchBlood No, the support here is not yet complete. Currently it works only for LDAP users.

**Comment by ArchBlood** [comment]
> @ArchBlood No, the support here is not yet complete. Currently, it works only for LDAP users.

Good to know, then there is something to build on 😄
